presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.cpp

```diff
@@ -1075,39 +1107,34 @@ HivePrestoToVeloxConnector::toVeloxSplit(
   infoColumns.insert(
       {"$file_modified_time",
        std::to_string(hiveSplit->fileSplit.fileModifiedTime)});
-  return std::make_unique<velox::connector::hive::HiveConnectorSplit>(
-      catalogId,
-      hiveSplit->fileSplit.path,
-      toVeloxFileFormat(hiveSplit->storage.storageFormat),
-      hiveSplit->fileSplit.start,
-      hiveSplit->fileSplit.length,
-      partitionKeys,
-      hiveSplit->tableBucketNumber
-          ? std::optional<int>(*hiveSplit->tableBucketNumber)
-          : std::nullopt,
-      customSplitInfo,
-      extraFileInfo,
-      serdeParameters,
-      hiveSplit->splitWeight,
-      infoColumns);
+  auto veloxSplit =
+      std::make_unique<velox::connector::hive::HiveConnectorSplit>(
+          catalogId,
+          hiveSplit->fileSplit.path,
+          toVeloxFileFormat(hiveSplit->storage.storageFormat),
+          hiveSplit->fileSplit.start,
+          hiveSplit->fileSplit.length,
+          partitionKeys,
+          hiveSplit->tableBucketNumber
+              ? std::optional<int>(*hiveSplit->tableBucketNumber)
+              : std::nullopt,
```

Comment:
Is this always null if there is bucketConversion ? Would be great to know if they are related.

Reply:
It is always non-null if there is bucketConversion.  I have check in Velox but can add a check here as well

Reply:
Would be great to add it here as well.